deployer/deployer-services/src/main/java/io/hyscale/deployer/services/model/PodStatusCode.java

```diff
@@ -19,8 +19,10 @@
 import java.util.Map;
 
 public class PodStatusCode {
+    
+    private PodStatusCode() {}
 
-	public static Map<Integer, Signals> statusCodeVsMessage = new HashMap<>();
+	private static final Map<Integer, Signals> statusCodeVsMessage = new HashMap<>();
```

Comment:
Not necessary to be outside ! This can sit inside the Signals enum itself.
The method should be indicate of podstatus code.